components/Form.tsx

```diff
@@ -1,8 +1,47 @@
+"use client"
+
 import Image from "next/image";
 import DogImage from "../public/Form images/dog_form.png";
 import CatImage from "../public/Form images/cat_form.png";
+import { FormEvent } from "react";
+import { useGoogleReCaptcha } from "react-google-recaptcha-v3";
 
 const Form = () => {
+  const { executeRecaptcha } = useGoogleReCaptcha();
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+
+    if (!executeRecaptcha) {
+      console.log("Not available to execute reCAPTCHA");
+      return;
+    }
+
+    const gRecaptchaToken = await executeRecaptcha("formSubmit");
+
+    try {
+      const response = await fetch("/api/route", {
```

Comment:
As mentioned in the route file comment, the API route should be `/api/recaptcha` for consistency